src/contracts/LANDAuction.ts

```diff
@@ -0,0 +1,15 @@
+import { Contract } from '../ethereum'
+import { fulfillContractMethods } from './verification'
+
+const { abi } = require('./artifacts/LANDAuction.json')
```

Comment:
If you use this TS declaration

```ts
declare module '*.json' {
  export const abi: any
}
```

You can then require JSON files as an import statement

```ts
import { abi } from './artifacts/LANDAuction.json'
```

Is not a review change but an observation only.